### ISSUE TITLE
Use developer ID for releases

### ISSUE THREAD

**Issue by orta** [uranusjr/macdown#434]
In WWDC 2015 Apple announced that they're making it free for people to install apps on their iOS devices. I think what this means is that you can get developer ID for free now, might be worth a second look for signing?

**Comment by uranusjr** [comment]
Yeah I saw that news too. Unfortunately, if I’m not mistaken, what Apple opens is *the ability to test iOS apps on a real device without a developer ID*. Developer IDs for OS X and iOS are not unified into one Apple developer ID, but it still costs $99 per year.

To break things down, developers now can:

* Test iOS apps without a developer ID on the iOS Simulator and real iOS devices. (Previously only possible with the Simulator.)
* Test and distribute OS X apps without a developer ID.
* Distribute iOS apps to the App Store with a developer ID.
* Distribute OS X apps to the App Store or independently with a developer ID.

So unfortunately the situation does not change for MacDown. :(

**Comment by uranusjr** [comment]
But this is certainly a problem. Maybe I should start a [Patreon](https://www.patreon.com/home) campaign. $99 is not *that* much.

**Comment by orta** [comment]
I'd chip in for sure.
